BrainCloudClient/Assets/BrainCloud/Client/BrainCloud/Internal/FileUploader.cs

```diff
@@ -111,9 +111,6 @@ public enum FileUploaderStatus
         {
             _client = client;
 
-#if UNITY_WEBPLAYER || UNITY_WEBGL
-            throw new Exception("File upload API is not supported on Web builds");
```

Comment:
This should not be removed, this is still the original call which still wants this check remember? We should throw an ecxception saying heyyyy webgl doesn't support loading from disk, refer to loading from memory call instead

Reply:
I may be thinking about the wrong spot, but make sure to mention what i mentioned above in the old call

Reply:
that was a secondary safety check as when the call is first used it will throw an exception if they are using a web build, so it never hit that anyways. I've also made sure that the exception mentions to use the new api instead in the case of a web build